client/components/dashboard/config/dashboard-config-model.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @copyright Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @fileoverview DashboardConfigModel encapsulates the settings for dashboards.
```

Comment:
Please use "for a dashboard" or similar assuming that this is per-dashboard individually. The plural makes it sound as if it's a shared setting across multiple dashboards. Unless I'm misunderstanding it, in which case I think it could use a clearer explanation. The design deck doesn't clarify this.